pkg/Library/NodeGraph/Runner.js

```diff
@@ -47,7 +47,7 @@ async removeStores({$stores}, state, service) {
   return Promise.all(entries($stores || {}).map(async ([storeId, store]) => {
     if (!store.connection) {
       delete state.stores[storeId];
-      await service({kind: 'StoreService', msg: 'RemoveStore', data: {storeId}});
+      await service({kind: 'RecipeService', msg: 'RemoveStore', data: {storeId}});
```

Comment:
Consider moving 'RemoveStore' to 'StoreService'?

Reply:
good point, followup PR coming right up